### PR TITLE
Controls updated

### PR DESCRIPTION
**Pull request by GloireIrenge**
This pull request comes with the following changes :
- Fixing the not working buttons
- updating the ui
- adding the not working image on the interface
- updating the README file

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,36 +3,42 @@
...
+                        Your browser does not support the audio element.
+                    </audio>
+                    <div class="controls">
+                        <button id="prev-button" class="btn btn-primary" onclick="prevSong()">&#9664;</button>
+                        <button id="play-pause-button" class="btn btn-primary" onclick="togglePlayPause()">&#9654;</button>
+                        <button id="next-button" class="btn btn-primary" onclick="nextSong()">&#9654;</button>
+                    </div>
+                    <div class="progress mt-3">
+                        <input type="range" id="progress-bar" value="0" min="0" max="100" step="1">
+                    </div>
```
Comment (mwafrika) [inline index.html]: Kindly use semantic tags to ensure a more meaningful, accessible and maintainable web.
Example: main, section, aside, article, ...

**Review by mwafrika — CHANGES_REQUESTED** [review]

**Comment by GloireIrenge** [comment]
Thank you so much, let me do it

**Review by mwafrika — APPROVED** [review]
### 🎊 🎊 💯 💯 Congratulations @GloireIrenge !!! your pull request is ready to be merged.

**Comment by GloireIrenge** [comment]
Thank you so much for the great news!

**Pull request merged**